src/Tizen.NUI.Scene3D/src/public/Controls/Model.cs

```diff
@@ -312,6 +312,33 @@ public Animation LoadBvhAnimation(string bvhFilename, Vector3 scale = null)
             return ret;
         }
 
+        /// <summary>
+        /// Load facial animation and assign to model.
+        /// facialFilename is json format what we predefined.
+        /// </summary>
+        /// <returns>Animaion of facial</returns>
+        // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Animation LoadFacialAnimation(string facialFilename)
+        {
+            global::System.IntPtr cPtr = Interop.Model.LoadFacialAnimation(SwigCPtr, facialFilename);
+            Animation ret = Registry.GetManagedBaseHandleFromNativePtr(cPtr) as Animation;
+            if(ret == null)
```

Comment:
Would you add a space between `if` and `(`.
```
if (ret == null)
```

Reply:
Thank you. I make all files in `Tizen.NUI.Scene3D` formatting